File: 08.feetWet/04.Retrolambda/app/src/main/java/info/juanmendez/android/retrolambda/MainActivity.java
package info.juanmendez.android.retrolambda;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.util.Log;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        Button buttonHello = (Button) findViewById(R.id.buttonHello);

        buttonHello.setOnClickListener(v -> {
            Log.i( "Retrolambda", "Welcome to Paradise City!");
        });
    }
}

File: 09.database_connection/08.object_box/app/src/main/java/info/juanmendez/objectbox/ui/MainActivity.kt
package info.juanmendez.objectbox.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import info.juanmendez.objectbox.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        main_add_band.setOnClickListener {

        }

        main_add_song.setOnClickListener {
        }
    }
}